new-site/docs-site/content/.vuepress/public/images/typesense-dynamodb/typesense-dynamodb.svg

```diff
@@ -0,0 +1,77 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="1920" height="1080" viewBox="0 0 1920 1080">
```

Comment:
Just noticed this... Could you remove the Python logo from this image? Makes it a little confusing.